includes/payment-methods/Configs/Definitions/AffirmDefinition.php

```diff
@@ -0,0 +1,278 @@
+	 *
+	 * @return string[] Array of country codes
+	 */
+	public static function get_supported_countries(): array {
+		return [
+			Country_Code::UNITED_STATES,
+			Country_Code::CANADA,
+		];
+	}
+
+	/**
+	 * Get the payment method capabilities
+	 *
+	 * @return string[]
+	 */
+	public static function get_capabilities(): array {
+		return [
+			PaymentMethodCapability::BUY_NOW_PAY_LATER,
+			PaymentMethodCapability::REFUNDS,
+			PaymentMethodCapability::DOMESTIC_TRANSACTIONS_ONLY,
+		];
+	}
+
+	/**
+	 * Get the URL for the payment method's icon
+	 *
+	 * @param string|null $account_country Optional. The merchant's account country.
+	 *
+	 * @return string
+	 */
+	public static function get_icon_url( ?string $account_country = null ): string {
+		return plugins_url( 'assets/images/payment-methods/affirm-logo.svg', WCPAY_PLUGIN_FILE );
+	}
+
+	/**
+	 * Get the URL for the payment method's dark mode icon
+	 *
+	 * @param string|null $account_country Optional. The merchant's account country.
+	 *
+	 * @return string Returns regular icon URL if no dark mode icon exists
+	 */
+	public static function get_dark_icon_url( ?string $account_country = null ): string {
+		return plugins_url( 'assets/images/payment-methods/affirm-logo-dark.svg', WCPAY_PLUGIN_FILE );
+	}
+
+	/**
+	 * Get the URL for the payment method's settings icon
+	 *
+	 * @param string|null $account_country Optional. The merchant's account country.
+	 *
+	 * @return string
+	 */
+	public static function get_settings_icon_url( ?string $account_country = null ): string {
+		return plugins_url( 'assets/images/payment-methods/affirm-badge.svg', WCPAY_PLUGIN_FILE );
+	}
+
+	/**
+	 * Get the testing instructions for the payment method
+	 *
+	 * @param string $account_country The merchant's account country.
+	 * @return string HTML string containing testing instructions
+	 */
+	public static function get_testing_instructions( string $account_country ): string {
+		return '';
+	}
+
+	/**
+	 * Get the currency limits for the payment method
+	 *
+	 * @return array<string,array<string,array{min:int,max:int}>>
+	 */
+	public static function get_limits_per_currency(): array {
+		return [
+			Currency_Code::CANADIAN_DOLLAR      => [
+				Country_Code::CANADA => [
+					'min' => 5000,
+					'max' => 3000000,
+				],
+			],
+			Currency_Code::UNITED_STATES_DOLLAR => [
+				Country_Code::UNITED_STATES => [
+					'min' => 5000,
+					'max' => 3000000,
+				],
+			],
+		];
+	}
+
+	/**
+	 * Whether this payment method is available for the given currency and country
+	 *
+	 * @param string $currency The currency code to check.
+	 * @param string $account_country The merchant's account country.
+	 *
+	 * @return bool
+	 */
+	public static function is_available_for( string $currency, string $account_country ): bool {
+		if ( ! PaymentMethodUtils::is_available_for( self::get_supported_currencies(), self::get_supported_countries(), $currency, $account_country ) ) {
+			return false;
+		}
+
+		return true;
```

Comment:
Why not just like this?

```suggestion
		return PaymentMethodUtils::is_available_for( self::get_supported_currencies(), self::get_supported_countries(), $currency, $account_country ) );
```

Reply:
You're right, it can definitely be simplified. I was too eager with the copy/paste from the existing definitions. Done in eb6e90ac6